DFC.App.ContactUs.Services.EmailService/DFC.App.ContactUs.Services.EmailService.csproj

```diff
@@ -17,6 +17,7 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="3.1.9" />
+    <PackageReference Include="Notify" Version="2.9.0" />
     <PackageReference Include="SendGrid" Version="9.21.0" />
```

Comment:
Should we still have a reference to SendGrid now that we've moved to Notify?

Reply:
Removed